README.md

```diff
@@ -1,24 +1,86 @@
 # Purdue Laundry API
 
-Backend for the Purdue Laundry application. It scrapes data from the Purdue Laundry alert page and provides a nice API for accessing it from the app. 
+This is a backend for the Purdue Laundry application. It scrapes 
+data from the Purdue Laundry alert page and provides a nice API 
+for accessing it from the app. 
 
+# Running the Server with Docker
 
-## Contributing
```

Comment:
Make sure to reference contributing.md in this doc. Unless I missed it.

Reply:
do we need to mention it? are they not competent enough to see that the file exists in the root directory?